tests/utils.go

```diff
@@ -1434,6 +1434,17 @@ func DeletePV(os string) {
 	}
 }
 
+func DeleteAndWaitForVirtualMachineToDisappear(vmi *v1.VirtualMachineInstance) {
```

Comment:
We want to extract with time everything out of `utils` and not add to it more.
We will need to find a good place for this (e.g. https://github.com/kubevirt/kubevirt/pull/3949/commits/3389960435ee4017e42abe56962b51c61441a928).

For now I would just re-implement it like it is done in `cleanupVMI`.